### PR TITLE
Docsreview

### PR DESCRIPTION
**Pull request by TsuyoshiUshio**
I can't see any instruction related installation of Azure CLI 2.0 and login instruction for it.

### CODE REVIEW

**Comment by TsuyoshiUshio** [comment]
Also, I add one review. `kubectl` suddenly comes up.  I add instruction and links for it.

**Comment by TsuyoshiUshio** [comment]
I noticed, if I use `az acs kubernetes install-cli` it downloads old version. So I add some notice.

**Comment by TsuyoshiUshio** [comment]
Like this. Just use `--client-version` for this issue.
```
.\kubectl.exe version
Client Version: version.Info{Major:"1", Minor:"4", GitVersion:"v1.4.5", GitCommit:"5a0a696437ad35c133c0c8493f7e9d22b0f9b81b", GitTreeState:"clean", BuildDate:"2016-10-29T01:38:40Z", GoVersion:"go1.6.3", Compiler:"gc", Platform:"windows/amd64"}
Server Version: version.Info{Major:"1", Minor:"5", GitVersion:"v1.5.3", GitCommit:"029c3a408176b55c30846f0faedf56aae5992e9b", GitTreeState:"clean", BuildDate:"2017-02-15T06:34:56Z", GoVersion:"go1.7.4", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by dtzar** [comment]
Thanks for the review Tsuyoshi!  I'd like to defer all of the CLI setup to Task 1 Step 1 versus explicit walkthrough to avoid having to maintain those steps. ;) https://docs.microsoft.com/en-us/azure/container-service/container-service-kubernetes-walkthrough

**Comment by dtzar** [comment]
That said - the kubectl version option comment you added could be useful.  If you submit that PR to the public repo, I would approve that change :)

File: HOL1/HOL1_readme.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ az acs kubernetes install-cli
 az acs kubernetes get-credentials --resource-group=$RESOURCE_GROUP --name=$CLUSTER_NAME
 ```
 You need to set PATH environment for the command. By default, `/usr/local/bin/kubectl` is for a Linux or macOS system, `C:\Program Files (x86)\kubectl.exe` on Windows.
+If you get an old version of kubectl, you can download it using `--client-version` for `az acs kubernetes install-cli`.
```
Comment (dtzar) [inline HOL1/HOL1_readme.md]: > Note: If it is desired to have a newer version of kubectl, download it using the `--client-version` switch.  For instance `az acs kubernetes install-cli --client-version 1.6.2`.

**Pull request closed without merging**